api/src/main/java/org/openmrs/module/inventorypoc/delivernote/service/DeliverNoteServiceImpl.java

```diff
@@ -182,4 +183,10 @@ public DeliverNoteItem findDeliverNoteItemByLotAndTokenNumber(final String lotNu
 	public DeliverNoteItem findDeliverNoteItemByDrugAndLotNumber(final Drug drug, final String lotNumber) {
 		return this.deliverNoteItemDAO.findByDrugAndLotNumber(drug, lotNumber, false);
 	}
+	
```

Comment:
empty

Reply:
@ningosi , removed empty lines.  What is the IDE are you using for java projects? Actually I'm using eclipse and there I have configured some  "Save Actions"(code style)  auto-format the code( to remove empty lines, remove unused imports, to ident the code, etc)

Reply:
Am using IntelliJ, now that we have a licence for it, can we all use the same IDE? or you like eclipse